src/components/AdminDashboard/AdminDashboard.js

```diff
@@ -70,117 +68,76 @@ const AdminDashboard = () => {
     window.scrollTo(0, 0);
   }, [HAS_VISITED_BEFORE]);
 
+  // resets the selected incidents when switching from approved to unapproved page
+  useEffect(() => {
+    setSelectedIds([]);
+    setAllSelected(false);
+  }, [listType]);
+
   const modalHandler = () => {
     setShowModal(false);
   };
 
-  const lastPage = Math.ceil(unapprovedIncidents.length / incidentsPerPage);
-
   const oktaAxios = useOktaAxios();
 
-  // getting unapproved/pending incidents from the database
-  useEffect(() => {
-    getData(oktaAxios, setUnapprovedIncidents);
-  }, []);
+  const fetchIncidents = () => {
+    getApprovedIncidents(oktaAxios)
+      .then(res => setApprovedIncidents(res))
+      .catch(err => console.log(err));
 
-  // getting approved incidents from database
-  useEffect(() => {
-    oktaAxios
-      .get('/dashboard/incidents/approved')
-      .then(res => {
-        setIncidents(res.data);
-      })
-      .catch(err => {
-        console.log(err);
-      });
-  }, []);
+    getPendingIncidents(oktaAxios)
+      .then(res => setPendingIncidents(res))
+      .catch(err => console.log(err));
 
-  // getting form-responses from DS database
+    getFormResponses(oktaAxios)
+      .then(res => setFormResponses(res))
+      .catch(err => console.log(err));
+  };
+
+  // getting all incident data
   useEffect(() => {
-    axios
-      .get(
-        'http://hrf-bw-labs37-dev.eba-hz3uh94j.us-east-1.elasticbeanstalk.com/to-approve'
-      )
-      .then(res => {
-        setFormResponses(res.data);
-      })
-      .catch(err => {
-        console.log(err);
-      });
+    fetchIncidents();
   }, []);
 
-  // setting up pagination display on dashboard
-  useEffect(() => {
-    const start = incidentsPerPage * pageNumber - incidentsPerPage;
-    const newCurrentSet = unapprovedIncidents.slice(
-      start,
-      start + incidentsPerPage
-    );
-    setCurrentSet(newCurrentSet);
-  }, [pageNumber, incidentsPerPage, unapprovedIncidents]);
```

Comment:
I'm assuming this was just moved, but going to leave a comment here to come back to this in-case I don't see this functionality reproduced further down.

Reply:
The pagination stuff on AdminDashboard gets passed down a level or two, and then doesn't get used, because the AntTable component handles pagination entirely on its own.